lib/draw_badge_api.ts

```diff
@@ -39,66 +41,85 @@ export class DrawBadgeApi {
     this.handwave = handwave;
     return this.forceDrawing();
   }
+
+  drawCameraSizeChange(videoWidth: number, videoHeight: number) {
+    this.videoWidth = videoWidth;
+    this.videoHeight = videoHeight;
+    // console.log('it finally changed!!!!!!')
+    // console.log(videoWidth, videoHeight)
+    return this.forceDrawing();
+  }
 }
 
 // TODO: make sure the imageData scale and resize correctly based on window size.
 //       make need to make some more Zoom API calls to get user window size.
-function drawEverythingToImage(nametag: NameTagBadge, handWave: HandWaveBadge): ImageData {
+function drawEverythingToImage(nametag: NameTagBadge, handWave: HandWaveBadge, videoWidth: number, videoHeight: number): ImageData {
```

Comment:
nit: I'd probably make an interface to represent the "video resolution" rather than making people remember the order of the two numbers here.